getting-started/create-article-cli.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Write Your First Article
+short_title: Write Article
+---
+
+
+# Write Your First Article
+
+**Goal:** Create and edit your first article in Curvenote using the CLI.  
+
+
+
+## 1. Create a New Article
+
+From inside your project folder, run:
+
+```bash
+curvenote create article my-first-article
+```
+
+This will:
+
+- Create a new Markdown file in your `content/` directory.
+- Add default frontmatter (metadata) like `title` and `date`.
+- Link the article into your site navigation if your `curvenote.yml` is set up.
+
+Example folder after running the command:
+
+```
+content/
+├── index.md
+└── my-first-article.md
+```
```

Comment:
This command does not exist. You just create new articles in your preferred text editor.